CONTRIBUTING.md

```diff
@@ -11,39 +11,21 @@ This crate has some code paths that depend on Wasm Atomics, which has some prere
 - Cargo's [`build-std`].
 - The `atomics` and `bulk-memory` target features.
 
-Example usage:
-
-```sh
-# Installing Rust nightly and necessary components:
-rustup toolchain install nightly --target wasm32-unknown-unknown --component rust-src
-# Example `cargo build` usage:
-RUSTFLAGS=-Ctarget-feature=+atomics,+bulk-memory cargo +nightly build -Zbuild-std=panic_abort,std --target wasm32-unknown-unknown
-```
+These are set using [`rust-toolchain.toml`](./rust-toolchain.toml).
```

Comment:
Same as above, a `rust-toolchain.toml` would change the default, which is undesirable.